tools/cwrap/cwrap.py

```diff
@@ -33,7 +33,8 @@ class cwrap(object):
     FUNCTION_CALL_TEMPLATE = Template("$capture_result$cname($call_arg);")
 
     DEFAULT_PLUGIN_CLASSES = [ArgcountChecker, ConstantArguments, OptionalArguments,
-                              ArgumentReferences, BeforeAfterCall, ReturnArguments, GILRelease]
+                              ArgumentReferences, BeforeAfterCall, ReturnArguments,
+                              GILRelease, WrapDim]
```

Comment:
It's used only for tensors, don't add it to the default plugins. Add it to the used plugins in `setup.py`.